book/src/languages.md

```diff
@@ -61,6 +61,7 @@ These configuration keys are available:
 | `config`              | Language Server configuration                                 |
 | `grammar`             | The tree-sitter grammar to use (defaults to the value of `name`) |
 | `formatter`           | The formatter for the language, it will take precedence over the lsp when defined. The formatter must be able to take the original file as input from stdin and write the formatted file to stdout |
+| `append-error-node`   | Whether to append a `(ERROR) @error` capture to the language query. (defaults to `true`) |
```

Comment:
For the option name, I think `highlight-errors` or something similar would be more intuitive.

I'm also curious whether this should be

* language-specific vs editor-wide
* on/off by default

IMO the error highlighting is pretty noisy when you're typing out a block but I'm not sure how others feel.

Reply:
There doesn't seem to be a straightforward way to use the editor config. I could either pass it a parameter (requires changing a lot of calls), or find a way to reference the editor config within the highlighter config.

Reply:
Hmm yeah good point. Using language config is ok then I think 👍